.github/workflows/prototype-tests.yml

```diff
@@ -34,7 +34,7 @@ jobs:
         run: pip install --progress-bar=off --no-build-isolation --editable .
 
       - name: Install other prototype dependencies
-        run: pip install --progress-bar=off scipy pycocotools h5py iopath
+        run: pip install --progress-bar=off scipy pycocotools h5py iopath pytorchvideo
```

Comment:
Since we want to run the reference test in CI, we add `pytorchvideo` to the third party packages we install. Since it is a pure Python library, there is no need to install it with the other PyTorch packages like `torch` or `torchdata` above.

Reply:
Let's remove the dependency. Adding this is a bit of an overkill; I understand that this is on the tests only but pytorchvideo is a research library which breaks from time to time and also has a cyclical dependency on TorchVision. If it's absolutely needed to have a reference implementation, we could copy it over on our references scripts.

Reply:
It is optional as explained above, but I don't see a reason to not have it. I mean things like https://github.com/pytorch/vision/pull/6812#discussion_r1002437589 make it clear that there are subtle things that we need to watch out for when refactoring the kernel. Why not encode the "right" behavior in an unoptimized reference function?

Reply:
I've removed `pytorchvideo` in 82c0aeebd636c8bb75977e007b9bfb340246f5b8 and copy-pasted the implementation into the reference function.